exporter/containerimage/export.go

```diff
@@ -359,6 +360,12 @@ func (e *imageExporterInstance) Export(ctx context.Context, src *exporter.Source
 	}
 	resp[exptypes.ExporterImageDescriptorKey] = base64.StdEncoding.EncodeToString(dtdesc)
 
+	dtdesclist, err := json.Marshal(map[string]any{"list": descriptors})
```

Comment:
Why the "list" subkey?

Reply:
It's because of the unmarshalling enforces this way

https://github.com/moby/buildkit/blob/d09703ed9f28d0f400ee65033ee4ad0c2903147a/cmd/buildctl/build.go#L440

Reply:
Not sure what you mean. I mean `"containerimage.descriptors": []`

Reply:
You mean the base64 decoding of string? This could be detected as well. But if not then smth. like

```
"containerimage.index": {"descriptors" : []}
```
or `"containerimage.manifest" or smth.

Reply:
It is possible to do by changing:

```go
var raw map[string]interface{}

to

var raw interface{}
```

But that means that the CLI version and the BuildKit daemon have to be compatible otherwise it gets output as base64